src/util/verify-chromium-port.js

```diff
@@ -0,0 +1,52 @@
+import { createServer } from 'node:http';
+
+import { UnusablePortError } from '../errors.js';
+
+/**
+ * Determine if a port is available
+ * @param {number} port Port to test
+ * */
+async function isPortAvailable(port) {
+  return new Promise((resolve) => {
+    const server = createServer();
+    server.once('listening', () => {
+      server.close(() => resolve(true));
+    });
+    server.once('error', () => resolve(false));
+    server.listen(port);
```

Comment:
```suggestion
    server.listen(port, '127.0.0.1');
```

By default, `listen()` listens on `::` and/or `0.0.0.0`, which would enable external clients on the network to connect to the server. This may also trigger a firewall notification. This all is undesirable.

To avoid these issues, explicitly pass in the localhost IP, i.e. `127.0.0.1`.